src/main/kotlin/no/nav/syfo/identhendelse/kafka/KafkaIdenthendelseTask.kt

```diff
@@ -1,32 +1,21 @@
 package no.nav.syfo.identhendelse.kafka
 
-import kotlinx.coroutines.runBlocking
 import no.nav.syfo.ApplicationState
 import no.nav.syfo.Environment
-import org.apache.avro.generic.GenericRecord
-import org.apache.kafka.clients.consumer.KafkaConsumer
+import no.nav.syfo.kafka.launchKafkaTask
 
 const val PDL_AKTOR_TOPIC = "pdl.aktor-v2"
 
-fun consumeIdenthendelse(
+fun launchKafkaTaskIdenthendelse(
     applicationState: ApplicationState,
     environment: Environment,
     kafkaIdenthendelseConsumerService: IdenthendelseConsumerService,
 ) {
-    val kafkaConfig = kafkaIdenthendelseConsumerConfig(environment.kafka)
-    val kafkaConsumer = KafkaConsumer<String, GenericRecord>(kafkaConfig)
-
-    kafkaConsumer.subscribe(
-        listOf(PDL_AKTOR_TOPIC)
+    val consumerProperties = kafkaIdenthendelseConsumerConfig(environment.kafka)
+    launchKafkaTask(
+        applicationState = applicationState,
+        consumerProperties = consumerProperties,
+        topic = PDL_AKTOR_TOPIC,
+        kafkaConsumerService = kafkaIdenthendelseConsumerService,
     )
-    while (applicationState.ready) {
-        runBlocking {
-            if (kafkaConsumer.subscription().isEmpty()) {
-                kafkaConsumer.subscribe(listOf(PDL_AKTOR_TOPIC))
-            }
```

Comment:
Denne funksjonaliteten er der for å støtte retries uten å ta ned podden på Identhendelser:
Hvis det oppstår en feil, så skal man i catch-blokka unsubscribe og vente 60 sekunder. Derfor er det viktig at man i while-loopen også subscriber på nytt dersom man har unsubscribet.

Så vi må ha noe lignende i den nye `launchKafkaTask`, eller implementere retry på en annen måte 🤔

Reply:
Ah, stemmer, var ikke meningen å fjerne det 😳

Reply:
Lagt inn "re-subscribe" i [launchKafkaTask](https://github.com/navikt/ispersonoppgave/pull/131/commits/353d0cd7f90017f9351ba722e58f748032a4d977)